src/test/groovy/com/mechanitis/demo/spock/OrderingServiceSpec.groovy

```diff
@@ -0,0 +1,125 @@
+package com.mechanitis.demo.spock
```

Comment:
This should be place under the same package as the service class we are testing: `package com.synacy.gradprogram.spock.exercise;`